frontend/projects/upgrade/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts

```diff
@@ -44,6 +47,7 @@ export class EnrollmentOverTimeComponent implements OnChanges, OnInit, OnDestroy
   copyGraphData: IEnrollmentStatByDate[] = [];
   isInitialLoad = true;
   showLabelOfxAxis = true;
+  graphColorIndicators = [];
```

Comment:
Can you please add the datatype for this?

Reply:
Assigned datatype to this.